src/paymentservice/src/main/java/ca/bc/gov/open/pssg/rsbc/dps/paymentservice/CalculateSinglePayment.java

```diff
@@ -84,9 +85,9 @@ public SinglePaymentResponse singlepaymenturl(@RequestParam(value = "transType",
 			client = new BamboraClientImpl(new URL(hostedPaymentEndpoint), merchantId, hashKey,
 					Integer.parseInt(minutesToExpire));
 
-			URL response = client.calculateSinglePaymentURL(null, BamboraTransType.valueOf(transType), invoiceNumber,
+			URL response = client.calculateSinglePaymentURL(new SinglePaymentRequest(null, BamboraTransType.valueOf(transType), invoiceNumber,
 					Double.parseDouble(totalItemsAmount), 0L, 0L, approvedPage, declinedPage, errorPage, ref1, ref2,
```

Comment:
do we need to pass 0L as they are static ? could be defaulted in the object directly